var/spack/repos/builtin/packages/hugo/hugo-resources-0.53.json

```diff
@@ -0,0 +1,522 @@
+    "destination": "."
+  },
+  {
+    "name": "github.com/spf13/afero",
+    "git": "https://github.com/spf13/afero",
+    "tag": "v1.2.0",
+    "placement": "vendor/github.com/spf13/afero",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/spf13/cast",
+    "git": "https://github.com/spf13/cast",
+    "tag": "v1.3.0",
+    "placement": "vendor/github.com/spf13/cast",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/spf13/cobra",
+    "git": "https://github.com/spf13/cobra",
+    "tag": "v0.0.3",
+    "placement": "vendor/github.com/spf13/cobra",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/spf13/fsync",
+    "git": "https://github.com/spf13/fsync",
+    "commit": "12a01e648f05",
+    "placement": "vendor/github.com/spf13/fsync",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/spf13/jwalterweatherman",
+    "git": "https://github.com/spf13/jwalterweatherman",
+    "commit": "94f6ae3ed3bc",
+    "placement": "vendor/github.com/spf13/jwalterweatherman",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/spf13/nitro",
+    "git": "https://github.com/spf13/nitro",
+    "commit": "24d7ef30a12d",
+    "placement": "vendor/github.com/spf13/nitro",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/spf13/pflag",
+    "git": "https://github.com/spf13/pflag",
+    "tag": "v1.0.3",
+    "placement": "vendor/github.com/spf13/pflag",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/spf13/viper",
+    "git": "https://github.com/spf13/viper",
+    "tag": "v1.3.1",
+    "placement": "vendor/github.com/spf13/viper",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/stretchr/testify",
+    "git": "https://github.com/stretchr/testify",
+    "commit": "04af85275a5c",
+    "placement": "vendor/github.com/stretchr/testify",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/tdewolff/minify/v2",
+    "git": "https://github.com/tdewolff/minify",
+    "tag": "v2.3.7",
+    "placement": "vendor/github.com/tdewolff/minify/v2",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/tdewolff/parse/v2",
+    "git": "https://github.com/tdewolff/parse",
+    "tag": "v2.3.5",
+    "placement": "vendor/github.com/tdewolff/parse/v2",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/wellington/go-libsass",
+    "git": "https://github.com/wellington/go-libsass",
+    "commit": "c63644206701",
+    "placement": "vendor/github.com/wellington/go-libsass",
+    "when": "@0.53",
+    "destination": "."
+  },
+  {
+    "name": "github.com/yosssi/ace",
+    "git": "https://github.com/yosssi/ace",
+    "tag": "v0.0.5",
+    "placement": "vendor/github.com/yosssi/ace",
+    "when": "@0.53",
+    "destination": "."
+  }
+]
```

Comment:
Now I see why you wanted a directive for this...

Reply:
:muscle:

Reply:
I played around with adding a `when` keyword to the `include_resouces` directive thinking to minimize things that actually got pulled in, but it's hard to explain how it'd interact with the `when` keyword of the resources themselves....